ReactiveXComponent/Connection/IXCPublisherFactory.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System;
+
+namespace ReactiveXComponent.Connection
+{
+    public interface IXCPublisherFactory : IDisposable
```

Comment:
IDisposable is useless for a factory
To not set every class as IDisposable. By default, do nothing. Add IDisposable when required